### ISSUE TITLE
[Bug]: Unable to use background_choice other than "minecraft"

### ISSUE THREAD

**Issue by RelativeSure** [elebumm/RedditVideoMakerBot#1470]
### Describe the bug

When I have minecraft set at the background_choice option it works fine. 
But if I set gta, motor-gta or cluster-truck as the choice instead of minecraft, it fails

```
╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ Saving Text to MP3 files...                                                                                          │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
Saving... ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.5 it/s
Saved Text to MP3 files successfully.
╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ Downloading screenshots of reddit posts...                                                                           │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
Launching Headless Browser...
Skipping translation...
Downloading screenshots... ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 0:00:00
Screenshots downloaded Successfully.
╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ Finding a spot in the backgrounds video to chop...✂️
│
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
Moviepy - Running:
>>> "+ " ".join(cmd)
Moviepy - Command successful
Background video chopped successfully!
╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ Creating the final video 🎥                                                                                          │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
[17:31:04]  Video Will Be: 31 Seconds Long                                                            final_video.py:124
Moviepy - Building video assets/temp/zzez2d/temp.mp4.
MoviePy - Writing audio in tempTEMP_MPY_wvf_snd.mp4
MoviePy - Done.
Moviepy - Writing video assets/temp/zzez2d/temp.mp4

t:   0%|                                                                              | 0/961 [00:00<?, ?it/s, now=None]
╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│                                                                                                                      │
│             Sorry, something went wrong with this version! Try again, and feel free to report this issue at GitHub   │
│ or the Discord community.                                                                                            │
│                                                                                                                      │
│             Version: 3.0                                                                                             │
│                                                                                                                      │
│             Story mode: False                                                                                        │
│                                                                                                                      │
│             Story mode method: 1                                                                                     │
│                                                                                                                      │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
Traceback (most recent call last):
  File "/home/<username redacted>/repos/RedditVideoMakerBot/main.py", line 117, in <module>
    raise err
  File "/home/<username redacted>/repos/RedditVideoMakerBot/main.py", line 96, in <module>
    main(post_id)
  File "/home/<username redacted>/repos/RedditVideoMakerBot/main.py", line 55, in main
    make_final_video(number_of_comments, length, reddit_object, bg_config)
  File "/home/<username redacted>/repos/RedditVideoMakerBot/video_creation/final_video.py", line 265, in make_final_video
    final.write_videofile(
  File "<decorator-gen-55>", line 2, in write_videofile
  File "/home/<username redacted>/.local/lib/python3.10/site-packages/moviepy/decorators.py", line 54, in requires_duration
    return f(clip, *a, **k)
  File "<decorator-gen-54>", line 2, in write_videofile
  File "/home/<username redacted>/.local/lib/python3.10/site-packages/moviepy/decorators.py", line 135, in use_clip_fps_by_default
    return f(clip, *new_a, **new_kw)
  File "<decorator-gen-53>", line 2, in write_videofile
  File "/home/<username redacted>/.local/lib/python3.10/site-packages/moviepy/decorators.py", line 22, in convert_masks_to_RGB
    return f(clip, *a, **k)
  File "/home/<username redacted>/.local/lib/python3.10/site-packages/moviepy/video/VideoClip.py", line 300, in write_videofile
    ffmpeg_write_video(self, filename, fps, codec,
  File "/home/<username redacted>/.local/lib/python3.10/site-packages/moviepy/video/io/ffmpeg_writer.py", line 220, in ffmpeg_write_video
    for t,frame in clip.iter_frames(logger=logger, with_times=True,
  File "/home/<username redacted>/.local/lib/python3.10/site-packages/moviepy/Clip.py", line 472, in iter_frames
    frame = self.get_frame(t)
  File "<decorator-gen-11>", line 2, in get_frame
  File "/home/<username redacted>/.local/lib/python3.10/site-packages/moviepy/decorators.py", line 89, in wrapper
    return f(*new_a, **new_kw)
  File "/home/<username redacted>/.local/lib/python3.10/site-packages/moviepy/Clip.py", line 93, in get_frame
    return self.make_frame(t)
  File "/home/<username redacted>/.local/lib/python3.10/site-packages/moviepy/video/compositing/CompositeVideoClip.py", line 111, in make_frame
    f = c.blit_on(f, t)
  File "/home/<username redacted>/.local/lib/python3.10/site-packages/moviepy/video/VideoClip.py", line 527, in blit_on
    img = self.get_frame(ct)
  File "<decorator-gen-11>", line 2, in get_frame
  File "/home/<username redacted>/.local/lib/python3.10/site-packages/moviepy/decorators.py", line 89, in wrapper
    return f(*new_a, **new_kw)
  File "/home/<username redacted>/.local/lib/python3.10/site-packages/moviepy/Clip.py", line 93, in get_frame
    return self.make_frame(t)
  File "/home/<username redacted>/.local/lib/python3.10/site-packages/moviepy/video/compositing/CompositeVideoClip.py", line 111, in make_frame
    f = c.blit_on(f, t)
  File "/home/<username redacted>/.local/lib/python3.10/site-packages/moviepy/video/VideoClip.py", line 536, in blit_on
    pos = self.pos(ct)
  File "/home/<username redacted>/repos/RedditVideoMakerBot/video_creation/background.py", line 28, in <lambda>
    background_options[name][3] = lambda t: ("center", pos + t)
TypeError: can only concatenate str (not "numpy.float64") to str
```

### Reproduction Steps

Change background_choice from minecraft to something else, like gta, motor-gta or cluster-truck

### Expected behavior

It would render the video normally like when you set the background_choice to minecraft

### System Information

Operating System : Windows 11
Python version : Python 3.10
App version / Branch : latest

### Checklist

- [X] I have searched the open issues for duplicates.
- [X] I have shown the entire traceback, if possible.

**Comment by RelativeSure** [comment]
Running Ubuntu 22.04 WSL on Windows 11**

**Comment by OpenSourceSimon** [comment]
Fixed in development, will soon be in v3.0.1
